README.md

```diff
@@ -1,30 +1,40 @@
 <div align="center">
-<img  align="center" src="./docs/logo.png" alt="Exposed" width="315" /></div>
+
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="./docs/logo-dark.png" width="315">
+    <img alt="Exposed logo" src="./docs/logo-light.png" width="315">
+  </picture>
+
+</div>
 <br><br>
 
+<div align="center">
+
 [![JetBrains team project](https://jb.gg/badges/team.svg)](https://confluence.jetbrains.com/display/ALL/JetBrains+on+GitHub)
-[![Kotlinlang Slack Channel](https://img.shields.io/badge/slack-@kotlinlang/exposed-yellow.svg?logo=slack?style=flat)](https://kotlinlang.slack.com/archives/C0CG7E0A1)
+[![Slack Channel](https://img.shields.io/badge/chat-exposed-yellow.svg?logo=slack)](https://kotlinlang.slack.com/messages/exposed/)
```

Comment:
This slack url format opens the channel directly in the slack app browser client and is the format also used in Ktor's readme.